.github/workflows/rasa-install-cron-check.yml

```diff
@@ -7,6 +7,11 @@ name: Cron Test to Check Pip Installation
 on:
   schedule:
     - cron: "0 0 * * *"
+  pull_request:
+    branches:
+    - main
+    tags:
+    - '*'
```

Comment:
```suggestion
```
You have to remove it, otherwise, it won't run for your PR.

Reply:
Ah! That might've been it. Will try that now.

Reply:
Mhmm. The "cronjob" still does not appear.

Reply:
Oh, the workflow was disabled, can you re-push your changes?

Reply:
It seems to run just fine now.

https://github.com/RasaHQ/rasa/runs/4934904869?check_suite_focus=true

I'll reset everything so that it's just a cron job again. This 'ought to do the trick.